src/rust/ide/view/graph-editor/src/builtin/visualization/native/raw_text.rs

```diff
@@ -44,9 +44,9 @@ impl RawText {
 
     /// Constructor.
     pub fn new(scene:&Scene) -> Self {
-        let network = default();
-        let frp   = visualization::instance::Frp::new(&network);
-        let model = RawTextModel::new(scene);
+        let network = frp::Network::new("raw_text");
```

Comment:
"raw_text" does not suggest that this is visualization frp. (Compare with "js_visualization_instance")